### PR TITLE
Added new pages integrations and sites

### PR DESCRIPTION
**Pull request by davidtsong**

### CODE REVIEW

File: src/content/platform/sites/config.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
+
+# Config
```
Comment (adamschwartz) [inline src/content/platform/sites/config.md:2]: Rename "Configuration" and change filename, and remove the empty line above.

File: src/content/platform/sites/config.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+</Definitions>
+
+## wrangler.toml
+
+There are a few specific configuration settings for Workers Sites in your `wrangler.toml`:
+
+<TableWrap>
+
+| Key           | Value                                                                              | Example                          | Required |
```
Comment (adamschwartz) [inline src/content/platform/sites/config.md:37]: Needs formatting.

File: src/content/platform/sites/config.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+| ------------- | ---------------------------------------------------------------------------------- | -------------------------------- | -------- |
+| `bucket`      | The directory containing your static assets, path relative to your `wrangler.toml` | `bucket = "./public"`            | yes      |
+| `entry-point` | The location of your Worker script, default is `workers-site`                      | `entry-point = "./workers-site"` | no       |
+| `include`     | A list of gitignore-style patterns for files or directories in `bucket` you exclusively want to upload. | `include = ["upload_dir"]` | no |
+| `exclude`     | A list of gitignore-style patterns for files or directories in `bucket` you want to exclude from uploads. | `exclude = ["ignore_dir"]` | no |
+
+</TableWrap>
```
Comment (adamschwartz) [inline src/content/platform/sites/config.md:44]: Needs new line after.

File: src/content/platform/sites/config.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+To learn more about the optional `include` and `exclude` fields, visit [Ignoring Subsets of Static Assets](/tooling/wrangler/sites/#ignoring-subsets-of-static-assets).
+
+_Note: if your project uses [environments](/platform/environments), make sure to place `site` at the top level config._
+
+Example of a `wrangler.toml`:
+
+```toml
+account_id = "95e..."
```
Comment (adamschwartz) [inline src/content/platform/sites/config.md:52]: Add filename.

**Comment by davidtsong** [comment]
I think this should be good now

**Pull request merged**